### PR TITLE
add grpc package version 1.54.3 and 1.48.4

### PR DESCRIPTION
**Pull request by pgeler**
Specify library name and version:  grpc/1.54.3

update abseil, c-ares dependency to 1.19.1, protobuf to 3.21.12
https://github.com/conan-io/conan-center-index/pull/17580 can be closed as it based on the previous version of the recipe

starting grpc-1.55+ using a newer version of protobuf https://github.com/grpc/grpc/pull/32606, as well as using CONFIG package so that we would need to upgrade protobuf first #16098

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by pgeler** [comment]
@uilianries, @SSE4 sorry to mention you directly, but it's a month for the PR without feedback 😵‍💫

File: recipes/grpc/all/conandata.yml [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+    sha256: "e941752638a92f21e27566903e1eb9b4a6380d70ca5ce91116b4cff6cb8fe2f2"
+  "1.48.4":
+    url: "https://github.com/grpc/grpc/archive/v1.48.4.tar.gz"
+    sha256: "0c3faa83e39d4f1ab55fe1476362b9ac3b81632a46dce7fd4d50271bce816b53"
+  "1.47.5":
+    url: "https://github.com/grpc/grpc/archive/v1.47.5.tar.gz"
+    sha256: "bf47a9b6903f0b6ca77c4584601ea6ef93efdad905e20b78a23417a362bc2190"
+  "1.46.7":
+    url: "https://github.com/grpc/grpc/archive/v1.46.7.tar.gz"
+    sha256: "1f9cf306a79e9a76ff36f2f0563b72c84ab9f2592372a742a234f360ae733e54"
```
Comment (uilianries) [inline recipes/grpc/all/conandata.yml:16]:
```suggestion
```
I would suggest by removing older versions, as this project is big and requires a good amount of CI time. It's allowed to remove old versions: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-for-removing-older-versions-of-a-package

Comment (pgeler) [reply recipes/grpc/all/conandata.yml]: 👍 make sense, keeping 1.48 as latest in 1.4x.x branch, it has some behavior changes between 1.4x.x and 1.5x.x

File: recipes/grpc/all/conandata.yml [hunk 2]
```diff
@@ -1,24 +1,12 @@
...
-    url: "https://github.com/grpc/grpc/archive/refs/tags/v1.46.3.tar.gz"
-    sha256: "d6cbf22cb5007af71b61c6be316a79397469c58c82a942552a62e708bce60964"
+  "1.54.3":
+    url: "https://github.com/grpc/grpc/archive/v1.54.3.tar.gz"
+    sha256: "17e4e1b100657b88027721220cbfb694d86c4b807e9257eaf2fb2d273b41b1b1"
+  "1.48.4":
+    url: "https://github.com/grpc/grpc/archive/v1.48.4.tar.gz"
+    sha256: "0c3faa83e39d4f1ab55fe1476362b9ac3b81632a46dce7fd4d50271bce816b53"
 patches:
-  "1.50.1":
```
Comment (uilianries) [inline recipes/grpc/all/conandata.yml:15]: please, don't forget to remove these patches files too.

**Comment by pgeler** [comment]
I'm about to give up, CI certainly has some issues 😢

**Review by uilianries — APPROVED** [review]
LGTM

**Review by SSE4 — APPROVED** [review]

**Comment by SpaceIm** [comment]
Is it not excessive to remove maintenance of the previous latest version of grpc in conancenter?

**Comment by SpaceIm** [comment]
> Is it not excessive to remove maintenance of the previous latest version of grpc in conancenter? Usually it's more progressive.

Bump, many recipes depend on 1.50.0 or 1.50.1

**Comment by uilianries** [comment]
@SpaceIm I'll open a PR adding them back. Thank you for reporting !

**Comment by uilianries** [comment]
@SpaceIm I created the PR https://github.com/conan-io/conan-center-index/pull/20743 to revert it.

**Pull request merged**